lsp-mode.el

```diff
@@ -7534,11 +7534,26 @@ such."
 
 (make-obsolete 'lsp-shutdown-workspace 'lsp-workspace-shutdown "lsp-mode 6.1")
 
+(defcustom lsp-auto-accept-workspace t
```

Comment:
nit: Should this be `lsp-auto-select-workspace`?
We have `lsp-restart` to automatically accept to restart workspace already. The functionality in this PR is more like automatically select workspace if there's only one.

Reply:
You are right, accept kind of implies a yes/no prompt. Fixed.